app/controllers/api/v1/cookie_and_jwt_api/protocol_subscriptions_controller.rb

```diff
@@ -11,6 +11,20 @@ def show
           render json: @protocol_subscription, serializer: Api::ProtocolSubscriptionSerializer
         end
 
+        # Specify either a start_date or a start_time. A given start_date takes precedence.
+        # If a start_time is specified (and no start_date is specified),
+        # the start_date is chosen as the given start_time added to midnight the next day.
+        def create
```

Comment:
Yep, but that was in the basic auth API, and this was here. There is some overlap, but that one is different because it requires a mentor, and every protocol subscription to be started with that mentor as mentor. The two methods share code (in particular, they call the same use case), so I thought it was ok like this.